### PR TITLE
Add ainWalletSigner

### PR DESCRIPTION
**Pull request by woomurf**
- Add ain wallet signer as window.ainetwork wrapper class.

### CODE REVIEW

File: src/signer/ain-wallet-signer.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import {Signer} from "./signer";
+
+export class AinWalletSigner implements Signer {
+  private ainetwork: Signer;
+  constructor() {
```
Comment (platfowner) [inline src/signer/ain-wallet-signer.ts:5]: minor) could add some empty lines between member variable definitions and method definitions?

File: src/types.ts [hunk 2]
```diff
@@ -1,4 +1,11 @@
 import { AxiosRequestConfig } from "axios";
+import {Signer} from "./signer/signer";
```
Comment (platfowner) [inline src/types.ts:2]: minor) some spaces like '{ Signer }'?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: src/signer/ain-wallet-signer.ts (hunk 1 above)
Comment (woomurf) [reply src/signer/ain-wallet-signer.ts:6]: Done!

**Comment by woomurf** [comment]
Thanks for the review!

**Pull request merged**